.circle/buildenv_st2.sh

```diff
@@ -51,6 +55,13 @@ re="\\b$DISTRO\\b"
 
 ST2_CIRCLE_URL=${CIRCLE_BUILD_URL}
 
-write_env ST2_GITURL ST2_GITREV ST2PKG_VERSION ST2PKG_RELEASE DISTRO TESTING ST2_CIRCLE_URL
+# NOTE: We don't build Mistral package on Ubuntu Bionic
```

Comment:
As code was moved to `buildenv_common.sh` now and can revert injecting this logic in `buildenv_st2.sh`?

Reply:
Done.